plugins/artifacts/config-schema.yml

```diff
@@ -0,0 +1,26 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: config
+description: Artifacts to be published
+type: array
+items:
+  type: object
+  properties:
+    type:
+      title: Upload type
+      description: Artifacts can be either an individual `file` or a `directory` containing potentially multiple files with recursively included subdirectories
+      type: string
+      enum: [file, directory]
+    localPath:
+      title: Local artifact location
+      description: Filesystem path of artifact
+      type: string
+    remotePath:
```

Comment:
`remotePath`/`name` should not start with `/`...

If type is folder it should end with `/`, and if type is file it should not ever end with `/`...

Also it should never contain any double slashes...